ConsentViewController/Classes/GDPRConsentViewController.swift

```diff
@@ -251,12 +251,19 @@ public typealias TargetingParams = [String:String]
 }
 
 extension GDPRConsentViewController: GDPRConsentDelegate {
+    @available(*, deprecated, message: "use gdprConsentUIWillShow() instead")
     public func consentUIWillShow() {
         loading = .Ready
         guard let viewController = messageViewController else { return }
         add(asChildViewController: viewController)
         consentDelegate?.consentUIWillShow?()
     }
+    
+    public func gdprConsentUIWillShow() {
+        guard let viewController = messageViewController else { return }
```

Comment:
@leodoin I believe you forgot to copy the following line
```swift
loading = .Ready
```

Reply:
indeed